src/nunit.gui.core/Model/Settings/GuiSettings.cs

```diff
@@ -81,9 +74,9 @@ public bool Maximized
         }
 
         private const string fontKey = "Font";
-        public Font Font
+        public ViewFont Font
```

Comment:
? I'm not sure what you mean by we can only store it as a string. With the TypeConverter added it gets saved and restored from this

```
<?xml version="1.0"?>
<NUnitSettings>
  <Settings>
    <Setting name="Gui.MainForm.Font" value="Microsoft Sans Serif;8.25" />
    <Setting name="Gui.MainForm.Location" value="10, 10" />
    <Setting name="Gui.MainForm.LocationAndSize" value="0, 0, 545, 454" />
    <Setting name="Gui.MainForm.Maximized" value="False" />
    <Setting name="Gui.MainForm.Size" value="1349, 613" />
    <Setting name="Gui.RecentProjects.File1" value="C:\Git\survey-design\mass-update\MassUpdate.AcceptanceTests\bin\Debug\MassUpdate.AcceptanceTests.dll" />
  </Settings>
</NUnitSettings>
```
and I can manually change it to say consolas size 14 and get a massivly large screen 

![image](https://cloud.githubusercontent.com/assets/3466221/26522835/5700a3f2-42d7-11e7-8fc9-b3da60f98a87.png)